_data/projects/raiden.yml

```diff
@@ -1,16 +0,0 @@
----
-name: Raiden Network
-desc: A proposed extension to Ethereum which scales-out asset transfer capacity in the network.
-site: https://github.com/raiden-network/raiden/
-tags:
-- blockchain
-- ethereum
-- payment-channels
-- microtransactions
-- state-channels
-upforgrabs:
-  name: good first issue
-  link: https://github.com/raiden-network/raiden/labels/good%20first%20issue
```

Comment:
I don't think we should delete this project - can you restore the file and update it to use this label:

https://github.com/raiden-network/raiden/labels/Tag%20%2F%20Good%20First%20Issue

Reply:
I am so sorry about my mistake. I didn't check the second pages of the tags.

Reply:
> We changed the label hacktoberfest of project 'home-assistant/home-assistant.io' from to Help want.
Should I change the link to
https://github.com/home-assistant/home-assistant.io/labels/Help%20wanted

Yes, see my suggested change above: https://github.com/up-for-grabs/up-for-grabs.net/pull/1694/files#r346884336